index.html

```diff
@@ -1032,7 +1039,15 @@ <h3>Concrete Lifecycle Example</h3>
   }
 }
         </pre>
-
+        <p>
+In this example, the <a>issuer</a> has bound the <a>verifiable credential</a> to
+the <a>holder</a> (Pat) using the
+<code>credentialSubject.id</code> property (rather than using the
+<code>holder.id</code> property). The second <code>proof</code> property in the
```

Comment:
```suggestion
In this example, the <a>issuer</a> has bound the <a>verifiable credential</a> to
the <a>holder</a> (Pat) using the
<code>credentialSubject.id</code> property (rather than using the
<code>holder.id</code> property). The second <code>proof</code> property in the
```
Aside from the continuing issue of inappropriate use of "binding", there is NOTHING in the existing doc (unless it snuck by me, which I hope it did not!) that says that the `credentialSubject.id` does double-duty as a `holder.id`, associating a VC with a specific Holder at issuance.  We spent many hours discussing this, and Holder-sameAs-Subject was neither determined to be the most-common nor the default scenario.  If anything, I remember us coming up with more Holder-**NOT**-sameAs-Subject scenarios.